io.github.rimsort.RimSort.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: io.github.rimsort.RimSort
+runtime: org.kde.Platform
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: "6.8"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: RimSort
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+  # Needed for steamcmd download
+  - --share=network
+
+  # XDG commons folders
+  - --filesystem=xdg-download
+
+  # Needed for autodetect config  /home/yodatak/.config/unity3d/Ludeon Studios/RimWorld by Ludeon Studios/Config
+  - --filesystem=~/.config/unity3d/Ludeon Studios/
+
+  # Used for reading SteamID of user
+  - --filesystem=~/.steam/
+  - --filesystem=~/.local/share/Steam
+  # Read paths to mods on a Steam Deck SD card
+  - --filesystem=/run/media
+
+  # to permit the workshop explorer to work
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+
+modules:
+  # generate from requirements.txt with python3 flatpak-pip-generator --requirements-file=requirements.txt --output pypi-dependencies
+  # # maturin
+  - maturin.json
+  # Python crypto
+  - python3-requirements-binaries.json
+  # Python dependencies generated from requirements.txt
+  - pypi-dependencies.json
+
+  - steamfiles.json
+
+  - steamworkspy.json
+
+  # add git cli tool
+  - name: git
+    make-args:
+      - INSTALL_SYMLINKS=1
+    make-install-args:
+      - INSTALL_SYMLINKS=1
+    cleanup:
+      - "/lib/pkgconfig"
+      - "/man"
+    sources:
+      - type: archive
+        url: https://www.kernel.org/pub/software/scm/git/git-2.48.1.tar.xz
+        sha256: 1c5d545f5dc1eb51e95d2c50d98fdf88b1a36ba1fa30e9ae5d5385c6024f82ad
+        x-checker-data:
+          type: anitya
+          project-id: 5350
+          stable-only: true
+          url-template: "https://www.kernel.org/pub/software/scm/git/git-$version.tar.xz"
+
+  - name: RimSort
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp -R app /app/bin/
+      - mkdir -p /app/bin/todds/
+      - chmod +x todds && cp todds /app/bin/todds/
+
+      - mkdir -p /app/bin/app/todds/
+      - chmod +x todds && cp todds /app/bin/app/todds/
+      - cp -R libs /app/bin/
```

Comment:
this doesn't look right, the prefix is `/app` not `/app/bin`

(applies to other things too)

Reply:
Thanks its done !